recipes/fc/build.sh

```diff
@@ -0,0 +1,8 @@
+
+#!/bin/bash
+set -xe
+#make CXX="${CXX}" INCLUDES="-I$PREFIX/include" CFLAGS+="-g -Wall -O2 -L$PREFIX/lib"
```

Comment:
You are not compiling anymore. Shipping binaries is not a good idea. Please try to compile it.